### PR TITLE
Section 3 updated and section 5

### PR DESCRIPTION
**Pull request by teckiegeek**
image

### CODE REVIEW

File: twindle-web/index.html [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="style.css" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (johnjacobkenny) [inline twindle-web/index.html:11]: Curious to know why we need jQuery, popper...

**Review by johnjacobkenny — COMMENTED** [review]
I'll leave this to others.

File: twindle-web/index.html (hunk 1 above)
Comment (proful) [reply twindle-web/index.html:11]: good catch Kenny. We don't need bootstrap dependency

**Review by proful — CHANGES_REQUESTED** [review]
Please remove bootstrap & jquery dependencies

**Comment by teckiegeek** [comment]
I did not know I cannot use bootstrap. I will remove it.

**Pull request closed without merging**